modules/related-posts/jetpack-related-posts.php

```diff
@@ -1,18 +1,19 @@
 <?php
+/**
+ * Jetpack_RelatedPosts class.
+ */
 class Jetpack_RelatedPosts {
 	const VERSION = '20150408';
 	const SHORTCODE = 'jetpack-related-posts';
-
```

Comment:
Don't remove these line breaks after the declaration of properties and the beginning of a PHPDoc block.